### ISSUE TITLE
Importing image files across yalc

### ISSUE THREAD

**Issue by deg-basis** [wclr/yalc#159]
I am trying coordinate several existing React projects by building a new library of shared components and resources.
I discovered Yalc a couple of days ago and, so far, it looks perfect for my needs -- maintaining a private library shared by a group of developers.

This is working fine for JavaScript files, e.g., React components. But, I've hit a problem with other files (.png images, so far).  I hope this is just something I've configured wrong...

Here's the story:  I've got the new library "horcrux", and a project using it "myApp".

In horcrux, `src/index.js` includes:

```
import ico_accept_button from "./img/fatcow/accept_button.png";
...
```

**Comment by wclr** [comment]
Yalc just copiest package content to your projects and makes some symlinks if (you use yalc link or yalc add --link). It is simple. Not sure what can go wrong in you setup.

**Comment by deg-basis** [comment]
Yeah, I know. I'm tearing out what little hair I have left trying to figure out what went wrong.
Can you possibly point me to any working examples of importing pngs (or other file assets) from a yalc distribution?
(if you can, and I can solve what went wrong, I'm happy to write up what I learn)

**Comment by deg-basis** [comment]
I've found some clues here
https://create-react-app.dev/docs/adding-images-fonts-and-files/
which I think explain why this is not working.  But, I don't know enough about CRA or Webpack to see how to get around it.

Clues there:
- [importing from an assets file] "ensures that when the project is built, webpack will correctly move the images into the build folder, and provide us with correct paths."
- "webpack finds all relative module references in CSS (they start with ./) and replaces them with the final paths from the compiled bundle"

It looks like there is magic in the build chain that causes webpack to copy the files, iff there is an import from the png asset file.

This is why things work in a standalone project, and why they work for me in Horcrux, and why yalc successfully copies the name-mangled png files.

But, it is also why the build of myApp does not copy the files. At that point, the imports look like they are of simple strings from Horcrux, so webpack has no clue that it needs to copy the asset files.

**Comment by wclr** [comment]
> magic in the build chain 

That's bad.

**Comment by deg-basis** [comment]
FWIW, this issue is unrelated to Yalc.

I followed up on my guess above, and now have a reproducible small case using just create-react-library and create-react-app.  See details (and, maybe someday, I hope, somebody's solution) at https://stackoverflow.com/questions/67266147/exporting-png-files-from-a-library-to-a-react-javascript-project
